net/node_test.go

```diff
@@ -262,7 +262,7 @@ func TestListenAddrs_WithListenP2PAddrStrings_NoError(t *testing.T) {
 	)
 	require.NoError(t, err)
 
-	require.Contains(t, n.ListenAddrs()[0].String(), "/ip4/127.0.0.1/tcp/")
+	require.Contains(t, n.ListenAddrs()[0].String(), "/tcp/")
```

Comment:
This test didn't actually exist before. If you look at the old node_test.go file, there is no such require contains. So that might be between 2 commits. It was passing locally but failing on the CI due to the IP being different every time.